### ISSUE TITLE
Feature Request: Pug support

### ISSUE THREAD

**Issue by lil5** [remoteinterview/zero#50]
It would be nice to work with pug files instead of HTML to get faster started with content.

<pugjs.org>

**Comment by asadm** [comment]
You are right. Pug support would be cool. I will look into this sometime if no one else does.
